### ISSUE TITLE
"Suggest new Coptic lemma" - user interface and automatic ID assignment

### ISSUE THREAD

**Issue by phoenix-mossimo** [KELLIA/dictionary#62]
> (amir-zeldes) That sounds like a good idea - would you be able to create a process for assigning future IDs for items that are not yet in the TLA? Would we need an ID region that is 'free' (so not yet used up for Coptic or earlier Egyptian lemmas, assuming some mapping already exists)? Or would new IDs be assigned as we go by upticking the maximal ID?

**Comment by amir-zeldes** [comment]
@phoenix-mossimo I meant more of an organizational process rather than literally auto-assigning IDs. Maybe just a contact e-mail explaining what information you'd want for a new submission? Since the dictionary DB gets wiped whenever we run the dictionary reader, the new entries would have to be put into XML. If you don't want to adopt them in TLA, maybe just having a separate XML file would work (the reader can read multiple files, as we used to do with one file per letter)

**Comment by dwerning** [comment]
Yes. We (will) have a process of quickly assinging new IDs based on (human) e-mail notice for the TLA for not just this project anyway. Need to discuss after release 2.0.0.
